### PR TITLE
feat(deployment): ensure usd values in deployments for managed wallets

### PR DESCRIPTION
**Pull request by ygrishajev**
image

### CODE REVIEW

File: apps/api/src/billing/routes/sign-and-broadcast-tx/sign-and-broadcast-tx.router.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ export const SignTxRequestInputSchema = z.object({
             "/akash.cert.v1beta3.MsgCreateCertificate",
             "/akash.market.v1beta4.MsgCreateLease",
             "/akash.deployment.v1beta3.MsgUpdateDeployment",
-            "/akash.deployment.v1beta3.MsgCloseDeployment"
+            "/akash.deployment.v1beta3.MsgCloseDeployment",
+            "/akash.deployment.v1beta3.MsgDepositDeployment"
```
Comment (baktun14) [inline apps/api/src/billing/routes/sign-and-broadcast-tx/sign-and-broadcast-tx.router.ts:20]: We'll re-add it to enable deposit for managed wallets, so might as well keep it?

**Review by baktun14 — COMMENTED** [review]
Perhaps we can have the same modifications in the deployment detail. Wdyt?

Actually, they're already shown in usd, but we could remove the tooltips that show in USDC.
image

File: apps/api/src/billing/routes/sign-and-broadcast-tx/sign-and-broadcast-tx.router.ts (hunk 1 above)
Comment (ygrishajev) [reply apps/api/src/billing/routes/sign-and-broadcast-tx/sign-and-broadcast-tx.router.ts:20]: I'm not sure I follow

Comment (baktun14) [reply apps/api/src/billing/routes/sign-and-broadcast-tx/sign-and-broadcast-tx.router.ts:20]: Oh my bad I must've been tired, I read it like you removed `MsgDepositDeployment`. Ignore my comment!

**Review by baktun14 — APPROVED** [review]

**Pull request merged**